airbyte-ci/connectors/metadata_service/orchestrator/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "orchestrator"
-version = "0.4.1"
```

Comment:
💅 We should run `poetry update metadata-service` to update the version in the lock file.

IIRC theres an issue with dagster deploy and caching if we dont do this...